### DISCUSSION TITLE
How to change wallets

### DISCUSSION THREAD

**Discussion by SwapnilSoni1999** [reown-com/appkit#3410] [Q&A]
Previously using modal opener 
if we call 

```js
open({ view: "Connect" })
```

It used to open connect modal and replaces the connection with newly switched account
but as of now it shows failed and clicking on try again does nothing.

**Comment by arein** [comment]
What are you building @SwapnilSoni1999 ? @rtomas can help

**Comment by rtomas** [comment]
Hi @SwapnilSoni1999 

nowadays to switch wallets, you need to disconnect and connect again.

What use case are you looking for?

**Reply by SwapnilSoni1999** [reply]
yeah realised lately I had an leaderboard application where users can connect and authorize multiple wallets in their logged in profile 
Implemented it finally but took really hard to implement that
